README.md

```diff
@@ -54,98 +64,98 @@ This default is fine for all configurations except `instadb` databases (you can
 
-## Overview: Managing a database ##
-
-* A database is created using `docker compose ... up ...`;
-* A database can be `stopped` _WITHOUT_ deleting its storage:
-  * `docker compose ... stop ...`;
-* A stopped database can be `restarted` and will _CONTINUE_ using its previous existing storage:
-  * `docker compose ... start ...`;
-* A database is `deleted` _WITH_ its storage using:
-  * `docker compose ... down`;
-* A client app connects to a database by configuring a port on the host network - set with the variable `EXTERNAL_ADDRESS` - into its connection string/params;
-  * example: `EXTERNAL_ADDRESS=192.167.0.123` if the local host's network address is `192.168.0.123`
-  * example: `EXTERNAL_ADDRESS=localhost` if `docker` can resolve 
-* Because `docker compose` does not scope the Docker networks it creates to a particular file or profile, `docker compose ... down` may attempt to delete a network that is still in use by a different database.
-The error looks like the following, and can be ignored:
-
-   ```
+### About Managing Databases ###
+
+- A database is created using `docker compose ... up ...`;
+- A database can be `stopped` _WITHOUT_ deleting its storage:
+  - `docker compose ... stop ...`;
+- A stopped database can be `restarted` and will _CONTINUE_ using its previous existing storage:
+  - `docker compose ... start ...`;
+- A database is `deleted` _WITH_ its storage using:
+  - `docker compose ... down`;
+- A client app connects to a database by configuring a port on the host network - set with the variable `EXTERNAL_ADDRESS` - into its connection string/params;
+  - example: `EXTERNAL_ADDRESS=192.167.0.123` if the local host's network address is `192.168.0.123`
+  - example: `EXTERNAL_ADDRESS=localhost` if `docker` can resolve 
+- Because `docker compose` does not scope the Docker networks it creates to a particular file or profile, `docker compose ... down` may attempt to delete a network that is still in use by a different database.
+  The error looks like the following, and can be ignored:
+
+   ```text
     ⠿ Network nuodb_net          Error                                                                                                                                                0.0s
     failed to remove network df0df85905b1702fea9c1a20a1142b9f4ff85f07844087b520f072c8a6af5e68: Error response from daemon: error while removing network: network nuodb_net id df0df85905b1702fea9c1a20a1142b9f4ff85f07844087b520f072c8a6af5e68 has active endpoints
    ```
 
 **WARNING:** In the following do not confuse `start` and `stop` with `up` and `down`.
-* `docker up` is used to create the database container and start it running.
-* You can stop and restart the container using `docker stop` and `docker start` - no database data will be lost.
-* `docker down` _destroys_ the container and your database will be lost.
+
+- `docker up` is used to create the database container and start it running.
```

Comment:
This is pretty old text.

We should probably change `docker up` to `docker compose ... up`, and likewise for the other commands in this section.

Reply:
Actually I think that's my bad. Should have said `docker compose`.

Reply:
> my bad
This, I think, is taking American English a step too far...